client/src/app/tabs/dmn/getDmnEditMenu.js

```diff
@@ -61,21 +60,14 @@ export function getDmnDrdEditMenu(state) {
 }
 
 export function getDmnDecisionTableEditMenu(state) {
-  const {
-    defaultCopyCutPaste,
-    defaultUndoRedo
-  } = state;
-
-  const undoRedoEntries = defaultUndoRedo
-    ? getDefaultUndoRedoEntries()

```

Comment:
Why would we need this?

Reply:
I tested whether there is a difference when having this when undo/redoing in direct editing mode, but I couldn't observe one. Also `state.defaultUndoRedo` will always be `true` (see [dmnEditor.js](https://github.com/camunda/camunda-modeler/blob/develop/client/src/app/tabs/dmn/DmnEditor.js#L364)). Therefore the `getDefaultUndoRedoEntries()` will run into the [Chromium bug ](https://github.com/electron/electron/issues/3682) IMO.

Reply:
I believe this code change is needed here: https://github.com/camunda/camunda-modeler/issues/1912